### PR TITLE
Typo fix: Runer -> Runner

### PR DESCRIPTION
**Pull request by soulshake**
Just a minor typo that slightly interfered with my `grep`s while working on https://github.com/integrations/terraform-provider-github/pull/821 :)

### CODE REVIEW

**Review by gmlewis — APPROVED** [review]
Thank you for catching that, @soulshake !
LGTM.

Awaiting second LGTM before merging.

**Review by willnorris — APPROVED** [review]

**Comment by gmlewis** [comment]
Thank you, @willnorris !
Merging.

**Pull request merged**
